content/blog/2019/07/2019-07-17-DevOps-World-Jenkins-World-2019-San-Francisco-Lunch-Demos.adoc

```diff
@@ -42,12 +42,15 @@ Discuss the security pitfalls of containers and how embedding an open source ima
 
 === Thursday  August 15, 2019
 
-==== 12:25 - 12:40pm	Results from the 2019 Jenkins Google Summer of Code	link:https://github.com/martinda[Martin d'Anjou]
+==== 12:25 - 12:35pm	Results from the 2019 Jenkins Google Summer of Code	link:https://github.com/martinda[Martin d'Anjou]
 
 In 2019, the Jenkins project participated in the Google Summer of Code. This is an annual, international, program which encourages college-aged students to participate in open source projects during the summer break between classes. In 2019, we had dozens of applications and many student projects. In this session, we will showcase the students' projects and talk about what they bring to the Jenkins ecosystem.
 
 
-==== 12:40 - 12:55pm		Sysdig Secure Jenkins Plugin		link:https://github.com/markyjackson-taulia[Marky Jackson]
+==== 12:35 - 12:45pm		Plugin installation CLI Tool		link:https://github.com/stopalopa[Natasha Stopa]
+This talk will demo the new plugin installation tool done as part of a Google Summer of Code project. It will show the CLI features and how the library has beenincorporatedinto other areas of Jenkins.
```

Comment:
```suggestion
This talk will demo the new plugin installation tool done as part of a Google Summer of Code project. It will show the CLI features and how the library has been incorporated into other areas of Jenkins.
```